packages/x-charts/src/ChartsVoronoiHandler/ChartsVoronoiHandler.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Delaunay } from 'd3-delaunay';
+import useEnhancedEffect from '@mui/utils/useEnhancedEffect';
+import { InteractionContext } from '../context/InteractionProvider';
+import { CartesianContext } from '../context/CartesianContextProvider';
+import { SVGContext, DrawingContext } from '../context/DrawingProvider';
+import { SeriesContext } from '../context/SeriesContextProvider';
+import { getValueToPositionMapper } from '../hooks/useScale';
+
+export type ChartsVoronoiHandlerProps = {
+  /**
+   * Defines the maximal distance between a scatter point and the pointer that triggers the interaction.
+   * If `undefined`, the radius is assumed to be infinite.
+   * @default undefined
```

Comment:
Have you considered going with a specific default? 🤔 
I checked that Highcharts also have the same behavior, but when exploring the demo, I felt that 25 seemed the most natural to me. 🤔 🤷 
What do others think? @flaviendelangle @joserodolfofreitas?

Reply:
Okay, if it's only my opinion, we can disregard it. There is nothing wrong with this default. 👌 
In that case, WDYT about aligning the demo with defaults?
It's what initially tripped me a bit because after seeing the demo I expected the default radius to be `50`. 🙈